### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -28,9 +28,11 @@ const outdir = 'dist';
 const packageJson = JSON.parse(fs.readFileSync('package.json'));
 
 const parser = (await import('argparse')).default.ArgumentParser();
+/* eslint-disable max-len */
 parser.add_argument('-r', '--rsync', { help: "rsync bundles to ssh target after build", metavar: "HOST" });
 parser.add_argument('-w', '--watch', { action: 'store_true', help: "Enable watch mode", default: process.env.ESBUILD_WATCH === "true" });
 parser.add_argument('-e', '--no-eslint', { action: 'store_true', help: "Disable eslint linting", default: lintDefault });
+/* eslint-enable max-len */
```
Comment (jelly) [inline build.js:35]: Splitting this up makes me really unhappy so I applied this instead. These changes need to go to starter-kit as well.

**Review by allisonkarlitskaya — DISMISSED** [review]
Big yes to the eslint config changes, thanks.

File: .gitignore [hunk 2]
```diff
@@ -11,6 +11,7 @@ Test*FAIL*
 /bots
 test/common/
 test/images/
+/test/static-code
 pkg
```
Comment (allisonkarlitskaya) [inline .gitignore:15]: I wonder if you also want to fix up the format of all of the surrounding things like → `pkg` → `/pkg/`

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+]
+line-length = 118
+src = []
+
+[tool.ruff.flake8-pytest-style]
+fixture-parentheses = false
+mark-parentheses = false
+
+[tool.ruff.isort]
+known-first-party = ["cockpit"]
```
Comment (allisonkarlitskaya) [inline pyproject.toml:44]: `cockpit` isn't a first-party to this module, and I don't expect that the integration tests would ever import anything from it anyway...

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+[tool.ruff]
```
Comment (allisonkarlitskaya) [inline pyproject.toml:1]: All that for a single test file?

Sure, I guess...

Comment (allisonkarlitskaya) [reply pyproject.toml:1]: fwiw, I was thinking about putting the `tsconfig.json` from cockpit into some common location (`pkg/lib` probably) where we could use it from other projects... I wonder if we want to do similar for our ruff config.

Comment (jelly) [reply pyproject.toml:1]: We already do this in podman, machines, with different selects so that will be some work.

File: .gitignore (hunk 2 above)
Comment (jelly) [reply .gitignore]: Done, according to podman's gitignore. This needs a fix in starter-kit.

**Review by allisonkarlitskaya — APPROVED** [review]
yup

**Pull request merged**